superbuild/ConfigureExternalProjects.cmake

```diff
@@ -11,6 +11,105 @@
 #
 ################################################################################
 
+## #############################################################################
+## Add common diffrent types of path for all external-projects
+## #############################################################################
+include(CMakeDependentOption)
+
+set(ep_prefix_comment      "A root directory used to compute other non-specified cmake paths of externals projects")
+set(ep_path_base_comment   "Specifies the common part of the build and source paths of externals projects")
+set(ep_dir_name_comment    "Directory name of external-projects")
+set(ep_path_source_comment "Sources path of the externals projects")
+set(ep_path_build_comment  "Sources path of the Externals projects")
+
+option(EP_CHECKBOX_CUSTOM_DIRS  "Allow to specify manually paths of externals projects " OFF)
+CMAKE_DEPENDENT_OPTION(EP_CHECKBOX_SIDE_BY_SIDE "Put build soure and build directories of externals projects side by side" OFF "NOT EP_CHECKBOX_CUSTOM_DIRS" OFF)
+CMAKE_DEPENDENT_OPTION(EP_CHECKBOX_ON_TOP_LEVEL "Put directory of externals projects at the same level of medInria" ON "NOT EP_CHECKBOX_CUSTOM_DIRS" OFF)
```

Comment:
Is it possible to remove "medInria" and put "the application" or something like that?

Reply:
I do this this but the goal is not to arase each trace of medInria but rather add MUSIC name or other custom mane in addition medInria.
I look forward to the finalization of the consortium for clear guidance on this.

Reply:
I will see David Margery at 10h30, I will ask him about the best strategy to adopt.